ch01.asciidoc

```diff
@@ -37,7 +37,7 @@ This is achieved by creating an economic incentive via a penalty mechanism if no
 Effectively the most recent update of the payment channel (encoded as a bitcoin transaction) becomes enforcable in this way. 
 // find / add sources for some of the claimes
 
-Already in July 2011 pseudonomous user hashcoin proposed on bitcointalk the usage of Timelocks via `nLockTime` to solve the custody problem of exchanges.
+In July of 2011, on the bitcointalk.org forum,  a pseudonomous user by the name of "hashcoin" proposed the usage of Timelocks via the `nLockTime` function of the bitcoin network to solve the custody problem of exchanges.
```

Comment:
What a surprise! hashcoin was the first one who proposed CoinJoin, too!

Bitcointalk, https://bitcointalk.org/index.php?topic=12751.msg315793#msg315793